src/screenshot/screenshot-compare.ts

```diff
@@ -168,6 +169,15 @@ async function getMismatchedPixels(pixelmatchModulePath: string, pixelMatchInput
         reject(err);
       });
 
+      pixelMatchProcess.on('exit', (code) => {
+        clearTimeout(tmr);
+        const exitError =
+          code === 0
+            ? new Error('Pixelmatch process exited unexpectedly')
```

Comment:
To make sure I understand this - pixelmatch exits with an error code of 0 when it ends unexpectedly? I would have expected '0' to signify a successful exit, which is why I ask

Reply:
From my understanding the forked process sends up a message to let the parent know it is done and can be killed, see:
```ts
pixelMatchProcess.on('message', (data: any) => {
  pixelMatchProcess.kill();
  clearTimeout(tmr);
  resolve(data);
});
```

Therefor I expect that any premature process exit is unexpected especially since I can't resolve the promise with any kind of result.

Reply:
I will add a comment to reflect this information in the code.